tardis/visualization/tools/sdec_plot.py

```diff
@@ -508,7 +508,7 @@ def from_hdf(cls, hdf_fpath, packets_mode=None):
                 }
             )
 
-    def _parse_species_list(self, species_list):
+    def parse_species_list(self, species_list):
```

Comment:
Is this being imported?

Reply:
yes

Reply:
does this serve a different functionality?